.github/ISSUE_TEMPLATE/exokit-desktop-bug-report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Exokit Desktop Bug report
+about: File an issue to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+ What happened, and what did you expect to happen?
+ 
+**To Reproduce**
+Steps or commands ran to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Additional context**
+Add any other context about the problem here.
+
+**System information:**
+ - OS: [e.g. Windows 10]
+ - Exokit Version [e.g. v0.0.512]
+ - Node version
+ - Npm version/build log if applicable
+ - Is your Exokit downloaded or built?
+
```

Comment:
Kinda inconsistent spacing here. Was this intentional?

Reply:
fixed in 815afce